src/ai-dialog-footer.ts

```diff
@@ -1,44 +1,46 @@
+import {inject} from 'aurelia-dependency-injection';
 import {customElement, bindable, inlineView} from 'aurelia-templating';
 import {DialogController} from './dialog-controller';
 
 /**
- * * View-model for footer of Dialog.
- * */
+ * View-model for footer of Dialog.
+ */
 @customElement('ai-dialog-footer')
+/* tslint:disable:max-line-length */
 @inlineView(`
   <template>
     <slot></slot>
-
     <template if.bind="buttons.length > 0">
       <button type="button" class="btn btn-default" repeat.for="button of buttons" click.trigger="close(button)">\${button}</button>
     </template>
   </template>
 `)
+/* tslint:enable:max-line-length */
+@inject(DialogController)
 export class AiDialogFooter {
-  static inject = [DialogController];
+  public static isCancelButton(value: string) {
+    return value === 'Cancel';
+  }
 
-  @bindable buttons: any[] = [];
-  @bindable useDefaultButtons: boolean = false;
+  @bindable()
```

Comment:
`@bindable public buttons: any[] = [];`

Should work here, was there an issue with the decorator here?